packages/state-transition/src/block/processWithdrawals.ts

```diff
@@ -121,8 +135,10 @@ export function getExpectedWithdrawals(
     }
   }
 
+  // partialWithdrawalsCount is withdrawals coming from EL since electra (EIP-7002)
   const partialWithdrawalsCount = withdrawals.length;
   const bound = Math.min(validators.length, MAX_VALIDATORS_PER_WITHDRAWALS_SWEEP);
+  const isElectra = fork >= ForkSeq.electra;
```

Comment:
I think the rest of the codebase use the term `isPostElectra`. Might want to be consistent with the naming

Reply:
I think we also need to guard off pre-capella from processing withdrawals. The guard is originally `isFullyWithdrawableValidator` and `isPartiallyWithdrawableValidator` but we removed them. Not 100% sure if this matters though

Reply:
if it's pre-electra then this function is not called at all, this is checked at the consumer side but I can add it in this function as well